src/test/v2/common/steps_file.ts

```diff
@@ -118,6 +118,7 @@ export = function() {
       this.runA11yCheck({ reportFileName: fileName });
       this.usePlaywrightTo('Run accessibility tests', async ({ page }) => {
         await injectAxe(page);
+        await checkA11y(page);
```

Comment:
Is that not what `this.runA11yCheck({ reportFileName: fileName });` does?

Reply:
I don't know, I asked Chand how to enable the fail on error check and he said its like this and hes already tested it. I am going to trust that it is this then.